packages/next/build/webpack/config/blocks/css/plugins.ts

```diff
@@ -2,36 +2,79 @@ import chalk from 'chalk'
 import { findConfig } from '../../../../../lib/find-config'
 import { resolveRequest } from '../../../../../lib/resolve-request'
 
-export async function getPostCssPlugins(dir: string): Promise<unknown[]> {
-  function load(plugins: { [key: string]: object | false }): unknown[] {
-    return Object.keys(plugins)
-      .map(pkg => {
-        const options = plugins[pkg]
-        if (options === false) {
-          return false
-        }
+type CssPluginCollection_Array = (string | [string, boolean | object])[]
 
-        const pluginPath = resolveRequest(pkg, `${dir}/`)
+type CssPluginCollection_Object = { [key: string]: object | boolean }
 
-        if (options == null || Object.keys(options).length === 0) {
-          return require(pluginPath)
-        }
-        return require(pluginPath)(options)
-      })
-      .filter(Boolean)
+type CssPluginCollection =
+  | CssPluginCollection_Array
+  | CssPluginCollection_Object
+
+type CssPluginShape = [string, object | boolean]
+
+const genericErrorText = 'Malformed PostCSS Configuration'
+
+function getError_NullConfig(pluginName: string) {
+  return `${chalk.red.bold(
+    'Error'
+  )}: Your PostCSS configuration for '${pluginName}' cannot have ${chalk.bold(
+    'null'
+  )} configuration.\nTo disable '${pluginName}', pass ${chalk.bold(
+    'false'
+  )}, otherwise, pass ${chalk.bold('true')} or a configuration object.`
+}
+
+async function isIgnoredPlugin(pluginPath: string): Promise<boolean> {
```

Comment:
```suggestion
function isIgnoredPlugin(pluginPath: string): Promise<boolean> {
```
Looks like this doesn't need to by `async`